### ISSUE TITLE
Copied?

### ISSUE THREAD

**Issue by coolpilot11** [coltonkalina/geofsliveryselector#3]
It appears to me that this code is copied from the Kolos liveryselector. I have notified him of it.

**Comment by kolos26** [comment]
Yes, It's has been copied from my LiverySelector project. But it's not a fork, the license and the copyright is missing. Please don't use the GeoFS LiverySelector under your name.
